src/test/shell/bazel/bazel_repository_cache_test.sh

```diff
@@ -494,18 +494,18 @@ EOF
     || echo "Expected fetch to succeed"
 }
 
-function test_experimental_repository_cache_urls_as_default_canonical_id() {
+function test_repository_cache_urls_as_default_canonical_id() {
   setup_repository
 
   bazel fetch --repository_cache="$repo_cache_dir" \
-        --experimental_repository_cache_urls_as_default_canonical_id \
+        --repository_cache_urls_as_default_canonical_id \
```

Comment:
Since the default flipped as well, should there be other tests that check the off behavior?

Reply:
Most functionality are the same, so I added a small test that essentially check for success instead of failure when the URL was updated.